src/pkg/packager/publish.go

```diff
@@ -0,0 +1,226 @@
+package packager
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"path/filepath"
+
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+
+	"oras.land/oras-go/v2"
+	"oras.land/oras-go/v2/content"
+	"oras.land/oras-go/v2/content/file"
+	"oras.land/oras-go/v2/registry/remote"
+	"oras.land/oras-go/v2/registry/remote/errcode"
+)
+
+// NOTES:
+// - This is a WIP, not yet functional
+// - This is a copy of the oras example code, with some modifications
+//
+// replace comments w/ CLI args where appropriate
+// replace fmt.Printf w/ message.Debug
+
+var zarfMediaType = "application/vnd.zarf.layer.v1+tar.zst"
+
+func (p *Packager) Publish() error {
+	registry := "localhost:666"
```

Comment:
can we change this to something like localhost:443 instead?

Reply:
was a holdover during transfer, disregard